server/odc-server/src/main/resources/data.sql

```diff
@@ -840,3 +840,8 @@ VALUES ('odc.permission-check.whitelist.database.ob-oracle',
             'dbms_trusted_certificate_manager, dbms_workload_repository, dbms_xplan',
         'Permission check whitelist for database in OB Oracle dialect type')
 ON DUPLICATE KEY UPDATE `id`=`id`;
+
+INSERT INTO config_system_configuration(`key`, `value`, `description`) VALUES('git.repository.preserve.minutes',
```

Comment:
`odc.integration.git.repository.preserve.minutes`
we may add a prefix~, same as below

Reply:
done